docs/guides/global-concurrency-limits.md

```diff
@@ -0,0 +1,182 @@
+---
+description: Learn how to control concurrency and apply rate limits using Prefect's provided utilities.
+tags:
+    - concurrency
+    - rate limits
+search:
+  boost: 2
+---
+
+# Global concurrency limits and rate limits
+
+Global concurrency limits allow you to manage task execution efficiently, controlling how many tasks can run simultaneously. They are ideal when optimizing resource usage, preventing bottlenecks, and customizing task execution are priorities.
+
+Rate Limits ensure system stability by governing the frequency of requests or operations. They are suitable for preventing overuse, ensuring fairness, and handling errors gracefully.
+
+When selecting between Concurrency and Rate Limits, consider your primary goal. Choose Concurrency Limits for resource optimization and task management. Choose Rate Limits to maintain system stability and fair access to services.
+
+## Managing Global concurrency limits and rate limits
+
+You can create, read, edit and delete concurrency limits via the Prefect UI. 
```

Comment:
I think this question is related to [chrisguidry's comment](https://github.com/PrefectHQ/prefect/pull/10496#pullrequestreview-1593944454) (a few weeks old now) -- is it the case that concurrency limits _need_ to be defined in the UI first before they can be used in the context managers? If that's the case, I think we can change `can` into a `must`.

Reply:
It's a little complicated since you _can_ just use it in your code and it'll be created for you, but in an inactive state.